Makefile

```diff
@@ -138,6 +150,19 @@ diff-check:
 tidy-deps:
 	go mod tidy
 
+.PHONY: $(GO_APP_BUILD_TARGETS)
+$(GO_APP_BUILD_TARGETS): build-%:
+	CGO_ENABLED=0 GOOS=$(GOOS) GOARCH=$(GOARCH) \
+		go build -ldflags "-s -w" \
+		-o bin/$* -mod=vendor cmd/$*/$*.go
+
+.PHONY: build-go-apps
+build-go-apps: $(GO_APP_BUILD_TARGETS)
```

Comment:
Because we use the prefix `build-%` [here](https://github.com/bucketeer-io/bucketeer/pull/23/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R154), I think it is better if match the prefix for other places.

E.g.
```sh
make build-go # Build all apps
make build-go-account # Build a single app
make test-go # Test all select directories
```

Reply:
Thx, I fixed it 👍